src/enums.h

```diff
@@ -82,6 +82,9 @@ enum {
     ENTRY_DISPATCHED,
     KEY_PRIVATE,
     ENTRY_FWD_HDR_WAIT,
+    /// Whether the entry serves collapsed hits now.
+    /// Meaningful only for public entries.
+    ENTRY_REQUIRES_COLLAPSING,
```

Comment:
These flags are written into disk cache (which persists across Squid releases). We should not change their values without a very good reason. I do not think we have such a reason here. Please move ENTRY_REQUIRES_COLLAPSING to the bottom of the `enum` list.

I doubt the current placement broke anything (i.e., the code probably "works") -- the flags below should not affect disk cache entries in the current Squid code, but that is not a good excuse to change their values.

Reply:
Fixed at 99755ec.